CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.27.4  (April 2024)
+
+- [Update]: Update OpenTok Android SDK and OpenTok iOS SDK to version 2.27.1.
```

Comment:
The previous version says Vonage Video iOS SDK and this one says Opentok SDK. Is that intentional?. Is this version update done in another PR, not seeing this version change in this PR.